src/lotus/tests.rs

```diff
@@ -0,0 +1,39 @@
+use url::Url;
+use crate::jsonrpc::JsonRpcClientImpl;
+use crate::lotus::{LotusClient, LotusJsonRPCClient};
+
+const HTTP_ENDPOINT: &str = "https://api.node.glif.io/rpc/v0";
```

Comment:
Let's hope that the endpoint doesn't change :) I don't have a better option without mocking the RPC API, so I am fine with this, but maybe let's add a note mentioning that if all tests fail the RPC API may no longer be available (which may be a common issue).

Reply:
Added comment for this. This parameter is also used in `jsonrpc` mod, I updated the code to use import instead, so that we only have one place to change.

Reply:
Yep. This is not ideal, but I think it's the best we can do for now. One option in the short-term, given the potential flakiness of these tests, is to mark them with `#[ignore]` and then explicitly run them with `cargo test -- --ignored`. But I think we can keep them as is for now.